run/hello-broken/main.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START run_broken_service]
+
+// Sample hello-broken demonstrates a difficult to troubleshoot service.
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+)
+
+func main() {
+	log.Print("hello-broken: service started")
+
+	http.HandleFunc("/", brokenHandler)
+
```

Comment:
Not sure how this whitespace will show up above this END and below the next START when run_broken_service is embedded. Might want to only have one of them with extra space in between.

Reply:
Done.